aucoalesce/normalize.go

```diff
@@ -52,7 +52,7 @@ type Strings struct {
 func (s *Strings) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	var singleValue string
 	if err := unmarshal(&singleValue); err == nil {
-		s.Values = append(s.Values, singleValue)
+		s.Values = []string{singleValue}
```

Comment:
This was causing bad anchor merge behavior. Basically, before this change, array sub-elements of an anchor node would get merged rather than overwritten when used, so:

```
macros:
  - &ecs-file
    category: file
    type: info

normalizations:
  - action: test
    ecs:
      <<: *ecs-file
      type: test
```

Would previously result in:

```
{
  ...
  "normalizations": [
    "action": "test",
    "ecs": {
      "category": ["file"],
      "type": ["info", "test"]
    }
  ]
}
```

rather than:

```
{
  ...
  "normalizations": [
    "action": "test",
    "ecs": {
      "category": ["file"],
      "type": ["test"]
    }
  ]
}
```

which is the intended (and to spec) result.